desdeo_tools/utilities/quality_indicator.py

```diff
@@ -37,22 +36,13 @@ def epsilon_indicator_ndims(reference_front: np.ndarray, front: np.ndarray) -> f
         float: The factor by which the approximating front is worse than the reference front with respect to all
         objectives.
     """
-
-    eps = 0.0
-    ref_len = reference_front.shape[0]
-    front_len = front.shape[0]
-    value = 0
-
-    for i in np.arange(ref_len):
-        for j in np.arange(front_len):
-            value = front[j] - reference_front[i][j]
-            if value > eps:
-                eps = value
-
-    return eps
+    eps_list = np.array(np.zeros(reference_front.shape[0]))
+    for i in np.arange(reference_front.shape[0]):
+        eps_list[i] = np.max(front - reference_front[i])
+    return eps_list
 
 
-def preference_indicator(reference_front: np.ndarray, front: np.ndarray, ref_point: np.ndarray, delta: float) -> float:
+def preference_indicator(reference_front: np.ndarray, front: np.ndarray, min_asf_value: float, ref_point: np.ndarray, delta: float) -> float:
```

Comment:
Similar comments here. You are just using these functions to compare two solutions. So do not call these variables "fronts" until you actually test them with fronts and they work as expected.